packages/relay-compiler-language-graphitation/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "devDependencies": {
     "@graphitation/graphql-js-tag": "^0.5.4",
+    "@graphql-tools/optimize": "^1.1.1",
```

Comment:
This will need to be a runtime dependency.

Reply:
Yeah, it's an oversight from me. Fixed now.